website/versioned_docs/version-29.7/GettingStarted.md

```diff
@@ -138,7 +138,7 @@ Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to
 Jest supports TypeScript, via Babel. First, make sure you followed the instructions on [using Babel](#using-babel) above. Next, install the `@babel/preset-typescript`:
 
 ```bash npm2yarn
-npm install --save-dev @babel/preset-typescript
+npm install --save-dev @babel/preset-typescript @babel/preset-env
```

Comment:
the thought was that you've already setup babel as mentioned above (`npm install --save-dev babel-jest @babel/core @babel/preset-env`), so in the TS case, you just need to add the TS preset. But maybe we should spell that out in the guide?

Reply:
> the thought was that you've already setup babel as mentioned above (`npm install --save-dev babel-jest @babel/core @babel/preset-env`), so in the TS case, you just need to add the TS preset. But maybe we should spell that out in the guide?

Yes, you're right. It is mentioned above.